src/hist/_internal/axis.py

```diff
@@ -1,22 +1,34 @@
+from __future__ import annotations
```

Comment:
```suggestion
```

Let's drop this one line and be 3.6+ compatible. There are a lot of 3.6 users out there. With end of life coming in September for 3.5, and NEP 29 already at 3.6+, we probably shouldn't bother with 3.5 support.

Reply:
It will make typing a little uglier down the line when we do fancier things, but mostly will work.